src/components/molecule_word_card/index.shared.tsx

```diff
@@ -34,35 +36,43 @@ const WordCardShared: FC<Props> = ({ wordId }) => {
     [wordId],
   )
 
-  if (sharedWord === undefined) return <WordCardSkeleton />
-  if (sharedWord === null || sharedWord.word === null)
+  if (sharedData === undefined) return <WordCardSkeleton />
```

Comment:
rollback the variable name from `sharedData` to `sharedWord` and create an issue with refactor that fix the naming problem we have

Reply:
OK. Handled in [2192054](https://github.com/ajktown/wordnote/pull/172/commits/21920546e52a48eee1106073782d08c2b037d830)